File: typescript/playground.js
//
// Typescript Playground
// 
// example 1 
var Greeter = (function () {
    function Greeter() {
    }
    Greeter.prototype.greet = function (name) {
        console.log(name);
    };
    return Greeter;
}());
var greeter = new Greeter();
greeter.greet('Hello from Typescript, Jungle Gym!');

File: swift/playground.swift
class Example {
	var name: String
	
	init(name: String) {
		self.name = name
	}

	func helloFriends() {
		let res = self.name
		print(res)
		return
	}
}

let example = Example(name: "Hello from Swift, Jungle Gym!")
example.helloFriends()